tutorials/analysis_3d.ipynb

```diff
@@ -947,6 +912,13 @@
     "\n",
     "* Analyse the second source in the field of view: G0.9+0.1 and add it to the combined model."
    ]
+  },
+  {
```

Comment:
This is an empty notebook cell...maybe remove?

Reply:
Done